reviews/2020-openebs-annual.md

```diff
@@ -0,0 +1,166 @@
+- Provide a framework to develop higher-level abstractions and operators for running stateful workloads like CockroachDB, Cassandra. 
+
+OpenEBS is focused on Storage engines and the standard Kubernetes abstractions to use them.  
+
+One of the ways in which users have benefited from the Rook and OpenEBS projects is to deploy the Stateful Workload using the Rook abstractions with OpenEBS storage classes.  
+
+Areas of future collaboration with the Rook community may include:
+- NFS provisioner
+- Application definitions
+- Further improvements to Kubernetes capabilities such as Block Device Management, Application Backup, Restore and Migration
+
+## Annual review Questions
+
+- *Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats.*
+
+  [OpenEBS DevStats](https://openebs.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m&from=now-1y&to=now-1h).
+  
+  Some  interesting trends:
+  - [Monthly release cadence since Aug 2019](https://openebs.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m&from=now-1y&to=now-1h).
+  - [Increase in average monthly commits (excluding docs)](https://openebs.devstats.cncf.io/d/2/commits-repository-groups?orgId=1&from=now-2y&to=now&var-period=m&var-repogroups=%22openebs%2Fapi%22&var-repogroups=%22openebs%2Fcharts%22&var-repogroups=%22openebs%2Fcstor%22&var-repogroups=%22openebs%2Fcstor-csi%22&var-repogroups=%22openebs%2Fcstor-operators%22&var-repogroups=%22openebs%2Fistgt%22&var-repogroups=%22openebs%2Fjiva%22&var-repogroups=%22openebs%2Fjiva-csi%22&var-repogroups=%22openebs%2Fjiva-operator%22&var-repogroups=%22openebs%2Flibcstor%22&var-repogroups=%22openebs%2Flinux-utils%22&var-repogroups=%22openebs%2Fmaya%22&var-repogroups=%22openebs%2FMayastor%22&var-repogroups=%22openebs%2Fnode-disk-manager%22&var-repogroups=%22openebs%2Fvelero-plugin%22&var-repogroups=%22openebs%2Fwebsite%22&var-repogroups=%22openebs%2Fzfs-localpv%22).
+  - [Tending towards 8/10 PRs closed per week](https://openebs.devstats.cncf.io/d/12/issues-opened-closed-by-repository-group?orgId=1&var-period=d7&var-repogroup_name=All). 
+  - [2 to 3 new contributors per month. Oct/Nov seeing the highest due to Hacktoberfest](https://openebs.devstats.cncf.io/d/52/new-contributors-table?orgId=1&from=now-1y&to=now).
+  - [10+ companies with 5+ contributions, and a total of 29 companies contributing in the last year](https://openebs.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=contributions).  
+  
+ - *How many maintainers do you have, and which organizations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)*
+   
+   OpenEBS source code, documentations and tests are distributed among several repositories. 
+   - There are a total of [7 maintainers](https://github.com/openebs/openebs/blob/master/MAINTAINERS) and
```

Comment:
I see that you have an issue open to consider moving the project to Incubation. While it's not an issue at the Sandbox stage, for incubation and graduation, the TOC will look for maintainer diversity. For those levels, the TOC wants to see a plan for (and evidence of) control and maintenance of the project and its code being shared among more than one organization.

Reply:
Added details(devstats link) on the contributing companies. I have made a note to expand on this in the Incubation PR.